### ISSUE TITLE
I think 1.5.0 breaks support for IIS virtual directories

### ISSUE THREAD

**Issue by Shazwazza** [dotless/dotless#458]
For example, if i have this request path:

http://localhost:5432/VirtualFolderTest/Css/MyLess.less

The response I get is:

/* File Not Found while parsing: You are importing a file ending in .less that cannot be found [/Css/SubFolder/myless.less]. */

and that is because IIS is configured to run in the virtual directory:

/VirtualFolderTest/

The dotless handler pre 1.5 handled this correctly.

**Comment by rytmis** [comment]
Thanks for reporting this. I'll check out why this happens and see if I can whip up a bugfix release soon-ish.

**Comment by rytmis** [comment]
OK, so I tried to run the current version of the web sample project under a virtual directory and it worked. Can you perhaps set up a minimal test case that reproduces the issue?

**Comment by lukeapage** [comment]
@Shazwazza do you have any imports that are inside rulesets or mixins ?

**Comment by Shazwazza** [comment]
Hi, no just regular imports. I'm sorry I haven't had time to re-verify this issue, i'll do my best to find some time this week.

**Comment by rytmis** [comment]
Thanks, if you could get me a simple repro project I could run myself, that would be awesome.

**Comment by rytmis** [comment]
@Shazwazza Do you think you might have the time to check for a repro soon-ish? I don't have a release schedule as such, but I'd love to get 1.5.1 out. :)

**Comment by Shazwazza** [comment]
Oh sorry!  Yes i have this on my TODO for tomorrow :)

**Comment by rytmis** [comment]
Awesome, thanks!

**Comment by Shazwazza** [comment]
Strangely, i could replicate the problem in my project. But then I created an empty sln just to verify this single issue and I had web.config errors:

     Cannot add duplicate collection entry of type 'add' with unique key attribute 'name' set to 'dotless'

So then I added this above the dotless handler before re-adding it:

     <remove name="dotless" />

and my new test project worked. Then i went back to my original project and started getting that web.config error... which wasn't there after I upgraded originally (very strange). So I added the same remove in the web.config and now it works again.

Not sure what was going wrong but I'll close the issue since it all seems to be working now. Might be good to update the web.config transform to add the remove before the add in the system.webServer/handlers section though.
